packages/validator/templates/1.0.0/endpoints.json

```diff
@@ -0,0 +1,71 @@
+{
+  "__arrayItem": {
+    "name": {},
+    "operation": {
+      "path": {
+        "__regexp": "^\\/[^\\s'\"\\\\]+$"
```

Comment:
OAS/Swagger allows for path templating. Is this something the current validator can handle? https://swagger.io/docs/specification/paths-and-operations/

Reply:
Absolutely, in api specifications path it even warns about missing parameter if it is not present in list of params of the path.